cli/build.gradle

```diff
@@ -36,6 +36,10 @@ tasks.register('downloadReleasesJson', Download)
 tasks.register('unzipManifest', Copy)
 tasks.register('unzipPortal', Copy)
 
+tasks.withType(Test).configureEach() {
+	jvmArgs += '--add-opens=java.base/java.lang=ALL-UNNAMED'
+}
```

Comment:
@kyle-miho you can do this conditionally based on the current JDK. See the [service builder plugin](https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-service-builder/src/main/java/com/liferay/gradle/plugins/service/builder/ServiceBuilderPlugin.java?plain=1#L238) for an example.

Reply:
For context: Java 8 does not like the `--add-opens` argument: https://github.com/liferay/liferay-blade-cli/actions/runs/15471791049/job/43557790701?pr=356#step:5:1184

Reply:
@kyle-miho I'd just go ahead and add all the opens statements from the linked file above. You're bound to run into this over and over if you just add them one at a time